ckanext/stadtzh_losdharvest/profiles.py

```diff
@@ -115,8 +116,6 @@ def parse_dataset(self, dataset_dict, dataset_ref):
         if publisher:
             dataset_dict["url"] = dataset_dict["author"] = publisher
 
-        dataset_dict["legalInformation"] = self._get_rights_for_dataset_ref(dataset_ref)
```

Comment:
If we're no longer using `_get_rights_for_dataset_ref`, can we get rid of that method please?

Btw, I guess you meant that `legalInformation` is now a string value, not `spatialRelationship`. :)

Reply:
Absolutely, you are right, legalInformation is now a string! IAnd I will remove the method. Thx for the review